### DISCUSSION TITLE
Incluir información de más bajo nivel (TLS)

### DISCUSSION THREAD

**Discussion by palmerabollo** [PucelaBits/websegura#133] [Ideas]
Cuando recorremos los sitios para preguntar al API de Mozilla, podríamos aprovechar y sacar los datos a nivel de TLS (versiones de TLS soportadas, cipher suites) para tenerlos y poder mostrarlos junto al informe técnico, o tener un ranking más inteligente que combine datos de mozilla (nivel de aplicación) con datos de bajo nivel.

Solo lo planteo como idea para discutir. No sé si es buena ya que el objetivo de la página no es tanto "maría, la informática friki" sino "manolo el panadero" y creo que no se pretende centrarse en detalles técnicos sino que sirva para presionar a los responsables públicos y que lo arreglen.

**Comment by ianvatega** [comment]
En esta web mira la versiónes tls

**Reply by palmerabollo** [reply]
¿A qué web te refieres? ¿A la de Mozilla? He visto que hay una pestaña TLS, pero es bastante lenta (creo que tira de ssllabs), del orden de minutos, y no he visto que haya API para consultar esos datos (quizá exista, pero no lo he encontrado).

**Reply by ianvatega** [reply]
Ostras, no la puse. ahora no la encuentro.. pero indicó esta otra a ver ...

[slabs](https://www.ssllabs.com/ssltest)

**Comment by nukeador** [comment]
El detalle técnico del observatory tiene una pestaña de scans externos que incluye eso. ¿Sospecho que algunos no tienen apis gratis?

Londe TLS puede ser útil cuando vuelvan los navegadores a dar un ultimátum: "Estas webs dejarán de funcionar en navegadores moderlos el DD/MM/YYYY"
